ethcore/res/ethereum/foundation.json

```diff
@@ -136,7 +136,8 @@
 				"eip100bTransition": "0x42ae50",
 				"difficultyBombDelays": {
 					"0x42ae50": "0x2dc6c0",
-					"0x6f1580": "0x1e8480"
+					"0x6f1580": "0x1e8480",
+					"0x8c6180": "0x3d0900"
```

Comment:
`0x8c6180`: 9200000
`0x3d0900`: 4000000

Reply:
Please review those two numbers against https://eips.ethereum.org/EIPS/eip-2384 and https://eips.ethereum.org/EIPS/eip-2387

Reply:
Numbers lgtm. 
It's a bit annoying that the EIPs are drafts and 2383 is not updated with the block number yet.

Reply:
https://github.com/ethereum/EIPs/pull/2408/files